CompuCell3D/core/Demos/CompuCellPythonTutorial/EnergyReport/README.txt

```diff
@@ -0,0 +1,9 @@
+This demo is incorrect and pending fixes. 
```

Comment:
The histograms should not show negative MCS. What should the axes show instead?
![342967663-3ac715cc-ae6e-435b-b490-30133cc963a8](https://github.com/CompuCell3D/CompuCell3D/assets/79017811/748c6076-9877-49a0-9512-046d4aba4502)

Reply:
The axis are just mislabeled. I'm assuming this code is in the demos at:
C:\CompuCell3D\Demos\CompuCellPythonTutorial\EnergyReport\EnergyReport.cc3d
Starting at line 30 in the steppables file (EnergyReportSteppables.py), the old lines are commented out and the new lines follow the old lines.
```
        self.plot_win3 = self.add_new_plot_window(
            title='Accepted Effective Energy Changes',
            #x_axis_title='Monte Carlo Step (MCS)',
            #y_axis_title='Effective Energy Change',
            x_axis_title='Effective Energy Change',
            y_axis_title='Count',
            grid=True)

        self.plot_win4 = self.add_new_plot_window(
            title='Rejected Effective Energy Changes',
            #x_axis_title='Monte Carlo Step (MCS)',
            #y_axis_title='Effective Energy Change',
            x_axis_title='Effective Energy Change',
            y_axis_title='Count',
            grid=True)
```

Reply:
Thanks so much for your insight, Jim. I will make these fixes.